support-frontend/assets/components/base/rows.jsx

```diff
@@ -4,7 +4,7 @@
 
 import React, { type Node } from 'react';
 
-import { classNameWithModifiers } from 'helpers/utilities';
+import { classNameWithModifiers } from 'helpers/utilities/utilities';
```

Comment:
I feel slightly suspicious on seeing `utilities/utilities` - could there be a more semantic name for this file?

Reply:
Yeah I'm not wild about some of the file names we currently, but I've tried to focus purely on moving files into folders in this PR- hopefully in a future PR we can sort out some of the names to make it clearer what they do, or sort out some of these big generic 'utilities' type files.